topics/introduction/slides/index.html

```diff
@@ -37,8 +37,14 @@
 
 ???
 
-- Information captured by Galaxy include: tool version, inputs and parameters.
-- Pages: interactive, web-based documents that describe a complete analysis.
```

Comment:
Any specific reason to remove the note about Pages? Can you restore it?

Reply:
Sorry, I deleted that by mistake. Thanks @nsoranzo!